apps/astarte_data_updater_plant/mix.exs

```diff
@@ -82,7 +83,7 @@ defmodule Astarte.DataUpdaterPlant.Mixfile do
       {:cyanide, "~> 2.0"},
       {:excoveralls, "~> 0.15", only: :test},
       {:mox, "~> 1.0", only: :test},
-      {:exandra, github: "vinniefranco/exandra"},
+      {:exandra, github: "vinniefranco/exandra", override: true},
```

Comment:
Let's use exandra 0.13 from hex.pm, which is at the same commit as master right now